buildengine/discover_test.go

```diff
@@ -1,69 +1,74 @@
 package buildengine
 
 import (
-	"context"
 	"testing"
 
 	"github.com/alecthomas/assert/v2"
 
 	"github.com/TBD54566975/ftl/common/moduleconfig"
-	"github.com/TBD54566975/ftl/internal/log"
 )
 
 func TestDiscoverModules(t *testing.T) {
```

Comment:
Add a test that includes an external library that imports one of the existing ftl modules in testdata.